js-exercises/freq-sort/freqSort.js

```diff
@@ -0,0 +1,25 @@
+function freqSort(arrayOfCharacters) {
+  if (!Array.isArray(arrayOfCharacters)) {
+    throw new Error("argument should be array");
+  } else {
+    let frequencyMap = {};
+
+    arrayOfCharacters.map(checkFreqElement);
+
+    function checkFreqElement(element) {
```

Comment:
Move function outside so it can be reused

Reply:
Also here I think function name can be improved as we are not checking frequency we are calculating it. Ex- getfrequencyMapOfArray not exactly but kind of